File: php-laravel/lib/app/Models/RequestTimeMapArrivalSearch.php
<?php
/**
 * RequestTimeMapArrivalSearch
 */
namespace app\Models;

/**
 * RequestTimeMapArrivalSearch
 */
class RequestTimeMapArrivalSearch {

    /** @var string $id */
    private $id;

    /** @var \app\Models\Coords $coords */
    private $coords;

    /** @var \app\Models\RequestTransportation $transportation */
    private $transportation;

    /** @var int $travelTime */
    private $travelTime;

    /** @var \DateTime $arrivalTime */
    private $arrivalTime;

    /** @var \app\Models\RequestTimeMapProperty[] $properties */
    private $properties;

    /** @var \app\Models\RequestRangeNoMaxResults $range */
    private $range;

    /** @var \app\Models\RequestLevelOfDetail $levelOfDetail */
    private $levelOfDetail;

}

File: fsharp-giraffe-server/OpenAPI/src/model/RequestTimeMapArrivalSearch.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open OpenAPI.Model.Coords
open OpenAPI.Model.RequestLevelOfDetail
open OpenAPI.Model.RequestRangeNoMaxResults
open OpenAPI.Model.RequestTimeMapProperty
open OpenAPI.Model.RequestTransportation

module RequestTimeMapArrivalSearch = 

  //#region RequestTimeMapArrivalSearch


  type RequestTimeMapArrivalSearch = {
    Id : string;
    Coords : Coords;
    Transportation : RequestTransportation;
    TravelTime : int;
    ArrivalTime : DateTime;
    Properties : RequestTimeMapProperty[];
    Range : RequestRangeNoMaxResults;
    LevelOfDetail : RequestLevelOfDetail;
  }
  //#endregion
  